### ISSUE TITLE
OpenStore 2 Downgrades manually installed packages

### ISSUE THREAD

**Issue by vanyasem** [UbuntuOpenStore/openstore-meta#150]
I have a newer version of Calendar installed (from a .click file) and OpenStore 2.00 suggests me to "downgrade" it. It installs the latest official version of this package (which in fact downgrades it).

My suggestions are: 
- Change the _color_ of the upgrade button to yellow (so I will realize this app requires my attention)
- Change the _text_ of the upgrade button to something like _"downgrade"_ or _"reinstall"_
- Add an option to "ignore" app updates for manually installed clicks

![photo_2017-09-22_14-47-05](https://user-images.githubusercontent.com/13350458/30743658-41ac7b5c-9fa7-11e7-875b-abeb361b5cb4.jpg)
**Installed:** 8.2
**Last avaliable:** 8.1

**Comment by TischlerWilly** [comment]
Is this the same issue? :
Openstore and system update have not the same app versions.
Example File Manager:
Version 0.4.528 in Open Store
Version 0.4.562 in system updates

**Comment by sverzegnassi** [comment]
Added an alert for applications that hasn't been installed from OpenStore. "Upgrade" button has been made less relevant. Fix will be available in v. 2.01, which will be released very soon.

@TischlerWilly It's similar to issue proposed here. UBports didn't released on OpenStore the version of File Manager that comes preinstalled with their system images.
Please follow this discussion: https://github.com/ubports/filemanager-app/issues/15

=======

In both cases, there is formally no bug in the OpenStore client. OpenStore is just one of the two app store services that are preinstalled in UBports images. We can only provide full support for those packages we are aware of their existance.

We can only smooth the behavior of the client-app when a package is installed by a different source; still, the client app should offer an option for installing the latest version available on the store, since it's considered the latest stable version available.

My personal opinion - the OS does not provide any tool for managing installed apps. All the major platforms (e.g. Android, iOS) provides some tool in their System Settings app for uninstalling apps, and keeps app store and system services very separated. On Ubuntu Touch this hasn't been made clear, and needs to be improved.
An app store can only handle its own applications, otherwise things might get hard to maintain. This has been somehow the case of the Ubuntu Store, which hasn't received relevant improvements for years.

Summing up - further improvements for such edge case might be provided in the near future. For now, I'll close this issue.

Thanks for your feedbacks!

**Comment by bqphone** [comment]
The new openstore update didnt fix the problem, still get the downgrade message from openstore for filemanager

**Comment by sverzegnassi** [comment]
@bqphone - The v2.01 update wasn't meant to fix the issue with File Manager. Unfortunately, there is nothing we can do for fixing your problem, since it's responsibility of the UBports team.

File Manager on OpenStore is owned by UBports: it's their duty to keep the OpenStore version in sync with the latest version available of their software.

We've been in touch with the UBports team, asking to upload the latest File Manager version. We have already taken every possible measure; we - as all of you - can only wait for them to fix the issue.

Sorry for the negative experience with the latest version of OpenStore app. We hope this will be solved in a very short time.

**Comment by bqphone** [comment]
Well thanks for clarifying, i did report it yesterday to ubports, i just thought that the update is related to that.

On the other hand the new openstore looks great, so great job.
